apps/emqx_lwm2m/src/emqx_lwm2m_coap_resource.erl

```diff
@@ -49,7 +49,7 @@
 
 % resource operations
 coap_discover(_Prefix, _Args) ->
-    [{absolute, "mqtt", []}].
+    [{absolute, "mqtt", {default,<<>>}}].
```

Comment:
this is a bug in spec.
should be: `-type coap_uri_param() :: [{atom(), binary()}].`